comment.py

```diff
@@ -14,7 +14,7 @@
     agent = CommentAgent(
         repo_name=REPO_NAME,
         branch_or_prnum=BRANCH_OR_PR_NUMBER,
-        branch=MAIN_BRANCH,
+        main_branch=MAIN_BRANCH,
```

Comment:
It's good practice to check the environment variable is set before using it to prevent `KeyError`. Implement checks or default values.